product_docs/docs/odbc_connector/12.2.0.2/01_odbc_release_notes.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Release Notes"
+
+legacyRedirectsGenerated:
```

Comment:
I don't think we want these generated redirects. They shouldn't be needed because this release was never on Docs 1.0.